### PR TITLE
Add Proprietary Codes for DSTU2

### PR DESCRIPTION
**Pull request by beichhor**

### CODE REVIEW

File: content/millennium/r4/proprietary-codes.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ This code set is similar to the HL7 value set but may have more specific display
 
     {
       "system": "https://fhir.cerner.com/<EHR source id>/codeSet/2,
-      "code": "309194
-      "display": "Emergency Room ",
+      "code": "309194"
```
Comment (beichhor) [inline content/millennium/r4/proprietary-codes.md]: Add comma

File: content/millennium/dstu2/proprietary-codes.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+# Proprietary Codes
+
+* TOC
+{:toc}
+
+## Overview
+
+Cerner's implementation of the HL7<sup>®</sup> DSTU2 FHIR<sup>®</sup> standard allows Millennium proprietary code values
+to be used in addition to standard value set codes on a limited basis. This allows developers to read and write data
```
Comment (MaxPhilips) [inline content/millennium/dstu2/proprietary-codes.md]: "on a limited basis." is distinct from the R4 page. not sure if we'd want to remove it there or add it here

File: content/millennium/dstu2/proprietary-codes.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            "system": "https://fhir.cerner.com/<EHR source id>/codeSet/14249",
+            "code": "21265426",
+            "display": "Same Day",
+            "userSelected": true
+          }
+        ]
+      }
+    }
+
+## Scheduling
```
Comment (MaxPhilips) [inline content/millennium/dstu2/proprietary-codes.md]: alphabetize Scheduling and Infrastructure

File: content/millennium/dstu2/proprietary-codes.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+<!-- use html header to avoid showing up in toc -->
+<h3>Slot</h3>
+
+The Slot Resource supports proprietary codes for:
+
+* Slot.type codes are maintained in [Code Set 14249 Scheduling Appointment Synonyms or Types](#code-set-14249-scheduling-appointment-synonyms-or-types)
+
+## Infrastructure
+
```
Comment (MaxPhilips) [inline content/millennium/dstu2/proprietary-codes.md]: I would prefer to keep the html header comment everywhere that we use one

File: content/millennium/dstu2/proprietary-codes.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+<h3>DocumentReference</h3>
+
+The DocumentReference Resource supports proprietary codes for:
+
+* DocumentReference.type codes are maintained in [Code Set 72](#code-set-72-event-code)
```
Comment (MaxPhilips) [inline content/millennium/dstu2/proprietary-codes.md]: Code Set 72 Event Code

File: content/millennium/dstu2/proprietary-codes.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+      "system": "https://fhir.cerner.com/<EHR source id>/codeSet/72,
+      "code": "2820587",
+      "display": "Discharge Note-Nursing",
+      "userSelected": true
+    }
+
+#### Code Set 14249 Scheduling Appointment Synonyms or Types
+
+This code set is highly customized by clients and may include appointment types such as
+Office Visit, New and Office, or Follow Up.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/proprietary-codes.md]: 'New and Office' is the example? or is that 'and' separating 'Office Visit, New' and 'Office'?

File: layouts/millennium_dstu2_sidebar.html [hunk 7]
```diff
@@ -9,6 +9,7 @@ <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></
           <li><a href="/millennium/dstu2/conformance/">Conformance Metadata</a></li>
           <li><a href="/millennium/dstu2/operation-definition/">OperationDefinition</a></li>
           <li><a href="/millennium/dstu2/structure-definition/">StructureDefinition</a></li>
+          <li><a href="/millennium/dstu2/proprietary-codes/">Proprietary Codes</a></li>
```
Comment (MaxPhilips) [inline layouts/millennium_dstu2_sidebar.html]: alphabetize

File: content/millennium/dstu2/proprietary-codes.md (hunk 2 above)
Comment (MaxPhilips) [reply content/millennium/dstu2/proprietary-codes.md]: talked offline, keeping this

File: content/millennium/dstu2/proprietary-codes.md (hunk 4 above)
Comment (beichhor) [reply content/millennium/dstu2/proprietary-codes.md]: https://github.com/cerner/fhir.cerner.com/pull/249/commits/9cfc80c04cad776c48bfa02d72a49e19ae3b016a#diff-e87f5cb37436794c3d11734635c2a942R52

File: content/millennium/r4/proprietary-codes.md (hunk 1 above)
Comment (beichhor) [reply content/millennium/r4/proprietary-codes.md]: https://github.com/cerner/fhir.cerner.com/pull/249/commits/9cfc80c04cad776c48bfa02d72a49e19ae3b016a#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R80

File: layouts/millennium_dstu2_sidebar.html (hunk 7 above)
Comment (beichhor) [reply layouts/millennium_dstu2_sidebar.html]: https://github.com/cerner/fhir.cerner.com/pull/249/commits/9cfc80c04cad776c48bfa02d72a49e19ae3b016a#diff-33cd408dcdc502f5079ff3f7117429e0L11

File: content/millennium/dstu2/proprietary-codes.md (hunk 6 above)
Comment (beichhor) [reply content/millennium/dstu2/proprietary-codes.md]: This should be clearer: https://github.com/cerner/fhir.cerner.com/pull/249/commits/9cfc80c04cad776c48bfa02d72a49e19ae3b016a#diff-e87f5cb37436794c3d11734635c2a942R92

File: content/millennium/dstu2/proprietary-codes.md (hunk 5 above)
Comment (beichhor) [reply content/millennium/dstu2/proprietary-codes.md]: https://github.com/cerner/fhir.cerner.com/pull/249/commits/9cfc80c04cad776c48bfa02d72a49e19ae3b016a#diff-e87f5cb37436794c3d11734635c2a942R57

File: lib/resources/dstu2/document_reference.yaml [hunk 8]
```diff
@@ -70,7 +70,9 @@ fields:
       note: Please note that the following is not the complete list of terminology bindings mapped for our sandbox. Our sandbox has many codes, but we only document a few to provide an example. This list also does not represent what would be mapped for one of our client’s domains. This is one of the things we evaluate and implement as needed when we are making apps available at client sites.
             <br/><br/>
             Please <a href="https://groups.google.com/d/forum/cerner-fhir-developers">contact us</a> if there is a code you are interested in consuming, and we can evaluate adding it to our sandbox.
-
+    - display: Millennium Event Code
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14249
+      info_link: https://fhir.cerner.com/millennium/dstu2/proprietary-codes/#code-set-72-event-code
```
Comment (jfrhong) [inline lib/resources/dstu2/document_reference.yaml]: can you make this a relative link

File: lib/resources/dstu2/slot.yaml [hunk 9]
```diff
@@ -21,3 +21,6 @@ fields:
     - display: Practice Setting Code Value Set
       system: http://snomed.info/sct
       info_link: http://hl7.org/fhir/dstu2/valueset-c80-practice-codes.html
+    - display: Millennium Scheduling Appointment Synonyms or Types
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14249
+      info_link: https://fhir.cerner.com/millennium/dstu2/proprietary-codes/#code-set-14249-scheduling-appointment-synonyms-or-types
```
Comment (jfrhong) [inline lib/resources/dstu2/slot.yaml]: can you make this a relative link

**Pull request merged**